compiler/rustc_lint/messages.ftl

```diff
@@ -224,9 +225,12 @@ lint_for_loops_over_fallibles =
 lint_forgetting_copy_types = calls to `std::mem::forget` with a value that implements `Copy` does nothing
     .label = argument has type `{$arg_ty}`
     .note = use `let _ = ...` to ignore the expression or result
+    .suggestion = use `let _ = ...` to ignore the expression or result
+
 lint_forgetting_references = calls to `std::mem::forget` with a reference instead of an owned value does nothing
     .label = argument has type `{$arg_ty}`
     .note = use `let _ = ...` to ignore the expression or result
+    .suggestion = use `let _ = ...` to ignore the expression or result
```

Comment:
Could you de-duplicate those messages, by making only one global. Something among the those lines.

```suggestion

lint_use_let_underscore_ignore = use `let _ = ...` to ignore the expression or result
```